composer.json

```diff
@@ -32,13 +32,19 @@
       "url": "https://wpackagist.org"
     }
   ],
+  "autoload": {
+    "psr-4": {
+      "Roots\\Bedrock\\": "lib/Bedrock"
```

Comment:
This shouldn't be necessary anymore now that Config has been moved to a package.

Reply:
whoops! I'm sloppy haha